jdaviz/configs/imviz/plugins/footprints/footprints.vue

```diff
@@ -78,86 +78,95 @@
       </div>
 
       <j-plugin-section-header>Footprint Definition</j-plugin-section-header>
-      <v-row>
-        <v-select
-          :menu-props="{ left: true }"
-          attach
-          :items="preset_items.map(i => i.label)"
-          v-model="preset_selected"
-          label="Preset"
-          hint="Select the preset instrument footprint."
-          persistent-hint
-        ></v-select>
-      </v-row>
+      <v-alert v-if="!has_pysiaf" type="warning" style="margin-left: -12px; margin-right: -12px">
+        <span>To use JWST footprints, install pysiaf and restart jdaviz</span>

```

Comment:
This is okay for now but I am not sure this is a good idea if we end up having multiple optional dependencies to support footprints for different telescopes.